backend/src/api/auth/auth.api.ts

```diff
@@ -45,6 +45,18 @@ const initAuthApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.status(HttpCode.OK).send(user);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.GET,
+    url: AuthApiPath.CURRENT_USER,
```

Comment:
```suggestion
    url: `$ApiPath.AUTH}${AuthApiPath.CURRENT_USER}`,
```
fix for previous also

Reply:
But we already have this as prefix in initApi.
![image](https://user-images.githubusercontent.com/35830708/183618692-d49e6563-6627-4853-9435-1ae842c696cb.png)
Should I remove it there and add it directly to url like in comment?

Reply:
oh sorry. got it

Reply:
resolved